cmd/server.go

```diff
@@ -185,7 +185,7 @@ func init() {
 	serverCmd.Flags().StringVar(&flagModuleArchiveFormat, "storage-module-archive-format", storage.DefaultModuleArchiveFormat, "Archive file format for modules, specified without the leading dot")
 
 	// Static auth options.
-	serverCmd.Flags().StringArrayVar(&flagAuthStaticTokens, "auth-static-token", nil, "Static API token to protect the boring-registry")
+	serverCmd.Flags().StringSliceVar(&flagAuthStaticTokens, "auth-static-token", nil, "Static API token to protect the boring-registry")
```

Comment:
Using `StringSliceVar` to allow passing multiple comma-separated tokens to the flag. `StringSliceVar` automatically splits the comma separates tokens :)